src/widgets/modalChildren/NewConfirmModal.js

```diff
@@ -45,7 +45,7 @@ export const NewConfirmModal = props => {
   if (modalProps.isOpen) Keyboard.dismiss();
 
   return (
-    <>
+    <Modal {...modalProps} style={style}>
```

Comment:
This change makes this component literally identical to the old component.

You should, for instance in `FirstUsePage.js`, be wrapping the `ConfirmModal` with `ModalContainer` in the parent component's render method. `ModalContainer` is in the `modals` folder. Search for "ModalContainer" in the project for examples.

Reply:
@Chris-Petty I couldn't find ModalContainer in FirstUsePage but instead it is wrapped with different types of containers like 
<View style={[globalStyles.verticalContainer, localStyles.verticalContainer]}>
It seems like database structure has been changed a bit. I will try to figure out where to get right Modal from since modal has been removed from react-native-modalbox in NewConfirmModal.

Reply:
In visual studio code press `cmd + p` and type in "ModalContainer" and select the option
![Jan-23-2020 12-11-52](https://user-images.githubusercontent.com/7684221/72943012-d2ad6d00-3dd9-11ea-85e8-bf29b44c3423.gif)